docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -114,7 +114,7 @@ endif::[]
 == Configuring the {product-kafka} instance for use with {product-long-connectors}
 
 [role="_abstract"]
-Configure your {product-kafka} instance for use with {product-long-connectors} by:
+After you create a {product-kafka} instance, configure it for use with {product-long-connectors} by:
```

Comment:
Per IBM style (and one of the topics in our MAS style discussions), introduce lists with a complete sentence.  

Suggest:

```
After you create a {product-kafka} instance, configure it for use with {product-long-connectors} by performing the following tasks:" 
```
And that would make the phrases that follow verb phrases:

"Create Kafka topics..."
"Create service accounts..."